src/Cardano/Prelude/HeapWords.hs

```diff
@@ -1,5 +1,5 @@
 {-# LANGUAGE FlexibleInstances #-}
-
+{-# LANGUAGE MagicHash #-}
```

Comment:
Can you keep the blank line before the module declaration please?
```suggestion
{-# LANGUAGE MagicHash #-}

```

Reply:
Sure, sorry about that.